### PR TITLE
 Bug 1123250 - Re-organize the lib files used for keyboard integration +autoland

### PR DESCRIPTION
**Pull request by RudyLu**
tests.
- Put system related lib to apps/system and user loader to load these
  lib files.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=16f1737fcabf)

**Comment by RudyLu** [comment]
Close to test autoland

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a9ad289f6ad7)

File: apps/keyboard/test/marionette/launch_test.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ marionette('show Keyboard APP', function() {
 
   setup(function() {
     keyboard =  new Keyboard(client);
-    system = client.loader.getAppClass('keyboard', 'system');
+    system = client.loader.getAppClass('system', 'input_management');
```
Comment (timdream) [inline apps/keyboard/test/marionette/launch_test.js]: `systemInputMgmt =`?

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d5e987a63ba0)

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#v80djt5OR7GjNSGl4sZobw

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1f0355be4839)

**Pull request merged**